CSharp/BatchExplorer/Models/Account.cs

```diff
@@ -28,7 +31,35 @@ public abstract class Account
         /// <summary>
         /// The key for the account.
         /// </summary>
-        public abstract string Key { get; set; }
+        public abstract string BatchServiceKey { get; set; }
+
+        /// <summary>
+        /// The service URL of the Linked Storage Account
+        /// </summary>
+        public abstract string LinkedStorageAccountName { get; set; }
+
+        /// <summary>
+        /// They access key of the Linked Storage Account
+        /// </summary>
+        public abstract string LinkedStorageAccountKey { get; set; }
+
+        private CloudBlobClient linkedStorageBlobClient;
+
+        public CloudBlobClient LinkedStorageBlobClient
+        {
+            get
+            {
+                if (linkedStorageBlobClient == null)
+                {
+                    if (!string.IsNullOrWhiteSpace(LinkedStorageAccountName) && !string.IsNullOrWhiteSpace(LinkedStorageAccountKey))
```

Comment:
minor: you could combine these two if conditions into one and save a level of indentation,